bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -340,17 +345,24 @@ impl<
 		match dest_para_id {
 			Some(dest_para_id) => {
 				let dest_para_fee_asset: Asset = (Location::parent(), dest_para_fee).into();
+				let bridge_location = Location::new(2, GlobalConsensus(network));
```

Comment:
I believe it is just a hypothetical problem if `dest_para_fee` and/or `asset` do not cover the ED on BH. Do we need to prefund this SA?

Reply:
yes, this has already been prefunded on live chains and is guaranteed to exist (satisfy ED)